uniflow/op/extract/split/recursive_character_splitter.py

```diff
@@ -0,0 +1,174 @@
+"""Recursive character split op."""
```

Comment:
nit: It can help ease the review process to make sure one PR is only for one feature in the future.